includes/definitions/aprisa.yaml

```diff
@@ -0,0 +1,14 @@
+os: aprisa
+text: 'Aprisa XE'
+type: wireless
+icon: 4RF
```

Comment:
Please lowercase the icon name here + lower case the actual file.

Same for group below.

Reply:
You will also need to lower case the mib dir.